test/test_helper.rb

```diff
@@ -1,2 +1,3 @@
 require "minitest/autorun"
+require 'minitest/pride'
```

Comment:
It gets you colours in the test output. Which I missed, and figured would be universally appreciated.